featuretools/primitives/standard/datetime_transform_primitives.py

```diff
@@ -459,6 +492,40 @@ def is_weekend(vals):
         return is_weekend
 
 
+class IsWorkingHours(TransformPrimitive):
+    """Determines if a datetime falls during configurable working hours
+
+    Args:
+        start_time (int): Start hour of workday. Default is 8 (8am)
+        end_time (int): End hour of workday. Default is 18 (6pm)
+
+    Examples:
+        >>> from datetime import datetime
+        >>> dates = [datetime(2022, 6, 21, 16, 3, 3),
+        ...          datetime(2019, 1, 3, 4, 4, 4),
+        ...          datetime(2022, 1, 1, 12, 1, 2)]
+        >>> iwh = IsWorkingHours()
+        >>> iwh(dates).tolist()
+        [True, False, True]
```

Comment:
Similar to above, but mention the inputs that are valid when customizing the hours and include a doctest example for customizing the values.